src/main/java/flight/reservation/plane/Aircraft.java

```diff
@@ -0,0 +1,22 @@
+package flight.reservation.plane;
+
+public class Aircraft {
+    public String model;
+    public int passengerCapacity;
+    public int crewCapacity;
+
+    public Aircraft(String model) {
+        this.model = model;
+    }
+
+    /*public String getModel() {
```

Comment:
Remove commented code blocks ;)

Reply:
In other words: We like the idea of adding the getter methods to the super class, but probably you lacked the time to do so...